src/content/blog/gpt2-mini-token-selection-visualization.md

```
@@ -0,0 +1,201 @@
+```
+
+重要なのは `softmax` と `multinomial` だ。
+
+- **softmax**: logits を確率分布（合計 1）に変換
+- **multinomial**: 確率分布に従ってランダムにサンプリング
+
+## 生成過程の実例
+
+「私は」というプロンプトで生成した結果を見てみよう。
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)